### ISSUE TITLE
Un-selected tabs - 508 contrast issue

### ISSUE THREAD

**Issue by gavinr** [Esri/calcite-react#366]
Unselected tabs do not have a sufficient contrast ratio

## To Replicate
1. Include CDN version of Calcite on a page
1. Include the tabs sample HTML from http://esri.github.io/calcite-web/documentation/patterns/#tabs (Example: https://jsbin.com/zegesov/edit?html,js,output)
1. Run the Google Chrome Developer Tools Audit (on https://output.jsbin.com/zegesov)
1. See the failing elements under `Accessibility > Contrast`
![image](https://user-images.githubusercontent.com/209355/78145607-9d984900-7420-11ea-9d8c-53c59430dd8f.png)

**Comment by gavinr** [comment]
Just to clarify here: to fix the 508 issue, I must use the updated release (v0.50.0) AND use the `translucent` version of the tabs?

**Comment by codylawson** [comment]
@gavinr yes. In the future the `translucent` style will be the default but right now we're just going to advise people who have compliance requirements to manually set it.

**Comment by gavinr** [comment]
sounds great, thanks!
